src/Arcus.Templates.Tests.Integration/WebApi/Swagger/v1/SwaggerDocExamplesTests.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Microsoft.OpenApi.Any;
+using Microsoft.OpenApi.Models;
+using Microsoft.OpenApi.Readers;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Arcus.Templates.Tests.Integration.WebApi.Swagger.v1
+{
+    [Collection(TestCollections.Integration)]
+    [Trait("Category", TestTraits.Integration)]
+    public class SwaggerDocExamplesTests
+    {
+        private readonly ITestOutputHelper _outputWriter;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="SwaggerDocExamplesTests"/> class.
+        /// </summary>
+        public SwaggerDocExamplesTests(ITestOutputHelper outputWriter)
+        {
+            _outputWriter = outputWriter;
+        }
+
+        [Fact]
+        public async Task ExampleProvidersNotIncluded_WithExcludeOpenApiDocs()
+        {
+            var options = new WebApiProjectOptions().WithExcludeOpenApiDocs();
+            using (var project = await WebApiProject.StartNewAsync(options, _outputWriter))
+            {
+                Assert.False(project.ContainsFile("ExampleProviders\\HealthReportResponseExampleProvider.cs"));
+            }
+        }
+
+        [Fact]
+        public async Task ExampleProvidersIncluded_WithoutExcludeOpenApiDocs()
+        {
+            var options = new WebApiProjectOptions();
+            using (var project = await WebApiProject.StartNewAsync(options, _outputWriter))
+            {
+                Assert.True(project.ContainsFile("ExampleProviders\\HealthReportResponseExampleProvider.cs"));
+            }
+        }
+
+        [Fact]
+        public async Task GetSwaggerDocs_ReturnsDocsWithHealthEndpointResponseExample()
+        {
+            // Arrange
+            var options = new WebApiProjectOptions();
+            using (var project = await WebApiProject.StartNewAsync(options, _outputWriter))
+            // Act
+            using (HttpResponseMessage response = await project.Swagger.GetSwaggerDocsAsync())
+            {
+                // Assert
+                Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+                string json = await response.Content.ReadAsStringAsync();
+                OpenApiDocument document = LoadOpenApiDocument(json);
+
+                var healthOperation = SelectGetHealthEndpoint(document);
+                
+                var okResponse = healthOperation.Responses
```

Comment:
```suggestion
                OpenApiResponse okResponse = healthOperation.Responses
```

Reply:
Changed this by pushing an additional commit